test/inductor/test_cpu_repro.py

```diff
@@ -999,6 +999,18 @@ def fn(x):
             (torch.randn(8),),
         )
 
+    def test_low_fp_index_expr_issue_147279(self):
+        # https://github.com/pytorch/pytorch/issues/147279
+        def fn(*args):
+            sym_0, sym_1, sym_2, sym_3 = args
```

Comment:
Small nit: suggest to use meaningful arg names instead of using `*args` and unpack them.

Reply:
Thanks for the comment and changed the args' name.